app/controllers/hyrax/downloads_controller.rb

```diff
@@ -105,5 +113,28 @@ def icon_path(asset_id, absolute_path = false)
         end
         ::ThumbnailPathService.icon_path(VisibilityService.visibility_value(file_set.rights_override))
       end
+
+      # Retrieve the content for relation
+      # @param [string] relation
+      # return [binary]
+      def relation_content(relation)
+        case relation
+        when 'preservation_master_file'
```

Comment:
Done in commit https://github.com/ucsdlib/horton/pull/129/commits/e8696d3a7ba2dc81566544144ec098610d5cdc12. Thanks.